app/components/text-currency/text-currency.tsx

```diff
@@ -45,7 +45,6 @@ export const TextCurrency: ComponentType = ({ amount, currency, style }: Props)
     <Text style={style}>
       {currency_fmt
         .default(amount, {
-          formatWithSymbol: false,
```

Comment:
dont we need the symbol? or is related to the "format" issue?

Reply:
`formatWithSymbol` is not accepted by this library. Perhaps it was deprecated? We don't want the symbol here as can be seen by `symbol: ""` below.